nni/common/graph_utils.py

```diff
@@ -725,8 +717,6 @@ def _build_graph(self):
 
         # associate module name with their trace graph nodes
         for node in graph.nodes():
-            if node.kind() == CONSTANT_KIND:
-                continue
```

Comment:
when we should skip the CONSTANT_KIND and when we should not? I found you did not remove all the `if node.kind() == CONSTANT_KIND:` in the code